Sources_v3/StreamChatUI/ChatChannel/Attachments/ChatFileAttachmentListView.swift

```diff
@@ -14,21 +14,15 @@ open class ChatFileAttachmentListView<ExtraData: ExtraDataTypes>: View, UIConfig
 
     public private(set) lazy var stackView: UIStackView = {
         let stack = UIStackView()
-        stack.isLayoutMarginsRelativeArrangement = true
-        stack.preservesSuperviewLayoutMargins = true
         stack.axis = .vertical
+        stack.spacing = 4
```

Comment:
system? 😱

Reply:
File attachment list view has all `margins == 4` as well as inter-item spacing in the [design](https://www.figma.com/file/ekifwChR9tR7zRJg1QEzSM/Q2_Stream_Mobile_Chat_v1?node-id=5243%3A34). With system spacing file list becomes clumsy